### DISCUSSION TITLE
I want to CONTRIBUTE to your projects named "[4] Collection and Study Management" in GSOC 23.

### DISCUSSION THREAD

**Discussion by SaikumarGatla** [camicroscope/GSOC#28] [General]
Hi! This is Saikumar Gatla- Third Year IT student at VPPCOE,Mumbai. I have knowledge about C++, Python, JavaScript, Reactjs. I am also familiar with numpy, pandas, matplotlib, seaborn, OpenCV, Tensorflow, Keras, Pytorch, Scikit-learn, Git and Github.

Recently I have made a small project Whatsapp Chat Analyzer(which creates a dataset from the chats and then show analysis in the form of graphs,piechart,wordcloud,heatmap,etc).

I can work on frontend as well as backend and I have a good understanding of machine learning and ML algorithms.

My Request is that:
(1)Can you tell me “Is there any channel or media through which I can communicate with you”.
(2)Can you provide me “Some Study Resources to Learn more about Your Projects”.

I have come here with an intension to learn and develop can you please point me to the right direction, as am I am a begineer in OPEN SOURCE and contributing first time in GSOC.
Eagerly waiting for your Reply. Thankyou !!!

**Comment by nanli-emory** [comment]
Hi @SaikumarGatla, I think the best way to start learning camicscope is to go through the camicscope repos and have a big picture of camicscope stack. install/deploy a Camic instance on your local and play with it.
